feedhenry/tests/org.jboss.tools.feedhenry.test/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.jboss.tools.aerogear.feedhenry</groupId>
+		<artifactId>tests</artifactId>
+		<version>1.0.0-SNAPSHOT</version>
+	</parent>
+	<groupId>org.jboss.tools.aerogear.feedhenry.tests</groupId>
+	<artifactId>org.jboss.tools.feedhenry.test</artifactId>
+	<packaging>eclipse-test-plugin</packaging>
+	
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-surefire-plugin</artifactId>
+				<configuration>
+					<explodedBundles combine.children="append">
+						<bundle>org.jboss.tools.feedhenry.test</bundle>
```

Comment:
It can be removed for now, if for some reason test bundles should be exploded after installation use 

``` shell
Eclipse-BundleShape: dir
```

in bundle's MANIFEST.MF file.

PR #20 removes that part of configuration from cordovasim and feedhenry tests.